### PR TITLE
add articles and update plan

### PR DESCRIPTION
**Pull request by JoeAAdams**

### CODE REVIEW

File: src/Components/ViewArticles.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import { newsApi } from "../App";
+import '../CSS/viewArticles.css'
+
+export default function ViewArticles () {
+    const [articles, setArticles] = useState([])
+    const [isLoading, setIsLoading] = useState([false])
+ 
+    useEffect(() => {
+        setIsLoading(true)
+        newsApi("/articles").then((response) => {
```
Comment (nc-bela-bertalan) [inline src/Components/ViewArticles.jsx:11]: Please extract the axios instance and API call into a separate file (utils/api.js).

File: src/CSS/viewArticles.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+}
+
+.article-item img {
+    height: 150px;
+    width: 150px;
+}
+
+.article-item h3 {
+    flex-grow: 1;
+}
```
Comment (nc-bela-bertalan) [inline src/CSS/viewArticles.css:27]: Please consider responsive design.

**Review by nc-bela-bertalan — COMMENTED** [review]
Hey Joe, please see my comments and amend your code.

File: src/CSS/viewArticles.css (hunk 2 above)
Comment (JoeAAdams) [reply src/CSS/viewArticles.css:27]: I'm not sure what responsive design could be done for this expect maybe hiding text overflow?

**Pull request merged**